mmv1/products/memcache/Instance.yaml

```diff
@@ -146,9 +146,10 @@ properties:
       The major version of Memcached software. If not provided, latest supported version will be used.
       Currently the latest supported major version is MEMCACHE_1_5. The minor version will be automatically
       determined by our system based on the latest supported minor version.
-    default_value: :MEMCACHE_1_5
```

Comment:
Changing default value in Terraform is a breaking change that we need to wait until major release to make such changes

Reply:
Set the default value to MEMCACHE_1_5.